Src/VsSpecific/Vs2017/Vs2017.csproj

```diff
@@ -8,7 +8,12 @@
     <DefineConstants>$(DefineConstants);VS_SPECIFIC_2017</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.Scripting, Version=2.10.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+    <Reference Include="Microsoft.CodeAnalysis, Version=2.10.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+    <Reference Include="Microsoft.CodeAnalysis.CSharp.Scripting, Version=2.10.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+    <Reference Include="Microsoft.CodeAnalysis.CSharp, Version=2.10.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />

```

Comment:
By using the 2.10 assemblies this code will only work on VS2017 update 9. Customers on VS2017 RTM will not be able to use this as they reference 2.0.0.0. That's a very small number of customers though. For a given version of Visual Studio only the RTM and latest update are supported. Overwelming majority of customers though are on the latest update. If anyone is one RTM and hits this we can re-evaluate. 

Commenting mostly in case anyone runs into this. Don't think it's an actual problem. We should leave as is.